### PR TITLE
Traypublisher: simple editorial multiple edl

### PR DESCRIPTION
**Pull request by jakubjezek001**
## Brief description
Traypublisher simple editorial supports multiple sequence files.

## Description
Multilayer edl files are now supported.

## Testing notes:
1. add multilayer edl files to sequence representations
2. add reviewable media
3. hit create and see nothing is broken

### CODE REVIEW

File: openpype/hosts/traypublisher/plugins/create/create_editorial.py [hunk 1]
```diff
@@ -239,35 +239,38 @@ def create(self, subset_name, instance_data, pre_create_data):
...
 
-        # create otio editorial instance
-        self._create_otio_instance(
-            subset_name, instance_data,
-            sequence_path, media_path,
-            otio_timeline
-        )
+            # create otio editorial instance
+            self._create_otio_instance(
+                subset_name + str(index),
```
Comment (iLLiCiTiT) [inline openpype/hosts/traypublisher/plugins/create/create_editorial.py:269]: This is just opinion: I don't think this is the right way but can't think of different solution. Also I would keep previous logic -> If only 1 edl is passed the subset name does not change.

Comment (iLLiCiTiT) [reply openpype/hosts/traypublisher/plugins/create/create_editorial.py]: This is resolved without any comment? This is change of logic whic his not backwards compatible, or?

**Comment by 64qam** [comment]
It fails when publishing: 
```DEBUG: {'active': True,
 'asset': 'editorial',
 'creator_attributes': {'Parent hierarchy path': 'shots/sq002',
                        'asset_name': 'sq002_sh010',
                        'clipDuration': 30,
                        'clipIn': 0,
                        'clipOut': 29,
                        'fps': 30.0,
                        'frameEnd': 1030,
                        'frameStart': 1001,
                        'handle_end': 0,
                        'handle_start': 0,
                        'sourceIn': 0,
                        'sourceOut': 30,
                        'workfile_start_frame': 1001},
 'creator_identifier': 'editorial_shot',
 'families': [],
 'family': 'shot',
 'hierarchy': 'shots/sq002',
 'id': 'pyblish.avalon.instance',
 'instance_id': '0477d1eb-8e42-4e48-8a19-8f5da7a7dcfb',
 'label': 'sq002_sh010_shotMain',
 'name': 'shotMain',
 'newAssetPublishing': True,
 'otioClip': '{\n'
             '    "OTIO_SCHEMA": "Clip.1",\n'
             '    "metadata": {\n'
             '        "cmx_3600": {\n'
             '            "reel": "sq002sh010"\n'
             '        }\n'
             '    },\n'
             '    "name": "sq002_sh010.mov",\n'
             '    "source_range": {\n'
             '        "OTIO_SCHEMA": "TimeRange.1",\n'
             '        "duration": {\n'
             '            "OTIO_SCHEMA": "RationalTime.1",\n'
             '            "rate": 30.0,\n'
             '            "value": 30.0\n'
             '        },\n'
             '        "start_time": {\n'
             '            "OTIO_SCHEMA": "RationalTime.1",\n'
             '            "rate": 30.0,\n'
             '            "value": 0.0\n'
             '        }\n'
             '    },\n'
             '    "effects": [],\n'
             '    "markers": [],\n'
             '    "media_reference": {\n'
             '        "OTIO_SCHEMA": "ExternalReference.1",\n'
             '        "metadata": {},\n'
             '        "name": "",\n'
             '        "available_range": {\n'
             '            "OTIO_SCHEMA": "TimeRange.1",\n'
             '            "duration": {\n'
             '                "OTIO_SCHEMA": "RationalTime.1",\n'
             '                "rate": 29.97002997002997,\n'
             '                "value": 3634.0\n'
             '            },\n'
             '            "start_time": {\n'
             '                "OTIO_SCHEMA": "RationalTime.1",\n'
             '                "rate": 29.97002997002997,\n'
             '                "value": 0.0\n'
             '            }\n'
             '        },\n'
             '        "target_url": '
             '"C:\\\\Users\\\\qam\\\\Downloads\\\\editorial_multilayer_test\\\\multilayer\\\\MER_sq002.mp4"\n'
             '    }\n'
             '}',
 'parents': [{'entity_name': 'qam_editorial', 'entity_type': 'Project'},
             {'entity_name': 'shots', 'entity_type': 'Folder'},
             {'entity_name': 'sq002', 'entity_type': 'Sequence'}],
 'publish_attributes': {},
 'representations': [],
 'shotName': 'sq002_sh010',
 'source': 'traypublisher',
 'subset': 'shotMain',
 'task': '',
 'tasks': {},
 'thumbnailSource': None,
 'timelineOffset': 0,
 'trackStartFrame': 0,
 'transientData': {},
 'variant': 'Main'}
Traceback (most recent call last):
  File "C:\Users\qam\code\OpenPype\.venv\lib\site-packages\pyblish\plugin.py", line 522, in __explicit_process
    runner(*args)
  File "C:\Users\qam\code\OpenPype\openpype\hosts\traypublisher\plugins\publish\collect_shot_instances.py", line 43, in process
  File "C:\Users\qam\code\OpenPype\openpype\hosts\traypublisher\plugins\publish\collect_shot_instances.py", line 84, in _get_otio_clip
IndexError: pop from empty list
```

**Comment by jakubjezek001** [comment]
> It fails when publishing:
Great catch! Yeah problem is that it is not merging those two EDLs together into one otioTimeline, which it should..

**Review by 64qam — APPROVED** [review]

**Pull request merged**